src/components/Version.js

```diff
@@ -25,16 +25,13 @@ class Version extends React.Component {
    * If it's software wallet show modal warning
    */
   walletTypeClicked = () => {
-    if (LEDGER_ENABLED) {
-      if (!LOCAL_STORE.isHardwareWallet()) {
-        $('#softwareWalletWarningModal').modal('show');
```

Comment:
This is not needed anymore, right?

Reply:
There are 2 changes here.
- Removing the `$('#softwareWalletWarningModal').modal('show');` since the modal uses the global modal component so this is not needed.
- Removing the `LEDGER_ENABLED` branch since it is hardcoded to true.

This `LEDGER_ENABLED` flag was created as a feature flag before we had a proper feature flag solution, but since it serves no actual purpose now and the flag was creating some confusion I decided to remove it.